static/src/javascripts/projects/commercial/sentinel.spec.ts

```diff
@@ -10,64 +10,144 @@ const config = (config_ as unknown) as {
 	get: jest.MockedFunction<(s: string, d: boolean) => boolean>;
 };
 
-jest.mock('lib/raven', () => ({
-	captureMessage: jest.fn(),
-}));
+MockDate.set(new Date('2021-01-01T00:00:00Z'));
+
+const CODE_ENDPOINT = '//logs.code.dev-guardianapis.com/log';
+const PROD_ENDPOINT = '//logs.guardianapis.com/log';
+const TEST_URL = 'http://testurl.theguardian.com/';
+
+const owner = 'commercial.sentinel';
+const mockDate = new Date();
+const defaultEvent: SentinelLoggingEvent = {
+	received_timestamp: mockDate,
+	received_date: mockDate.toISOString().slice(0, 10),
+	label: owner,
+};
 
 jest.mock('../../lib/config', () => ({
 	get: jest.fn(),
 }));
 
+beforeEach(() => {
+	navigator.sendBeacon = jest.fn();
+});
+
```

Comment:
Does this need to be mocked each time, or just once?

```suggestion
navigator.sendBeacon = jest.fn();

```

Reply:
Just once is enough!